Makefile

```diff
@@ -1372,9 +1372,6 @@ dataplane_kuttl_cleanup:
 .PHONY: dataplane_kuttl_prep
 dataplane_kuttl_prep: dataplane_kuttl_cleanup
 	$(eval $(call vars,$@,dataplane))
-	mkdir -p ${OPERATOR_BASE_DIR} ${OPERATOR_DIR}
-	pushd ${OPERATOR_BASE_DIR} && git clone ${GIT_CLONE_OPTS} $(if $(DATAPLANE_BRANCH),-b ${DATAPLANE_BRANCH}) ${DATAPLANE_REPO} "${OPERATOR_NAME}-operator" && popd
```

Comment:
These two lines are required because kuttl file should be present in the OPERATOR_BASE_DIR directory.